compliance/tags/tag_checker/tag_checker.pt

```diff
@@ -291,12 +291,41 @@ policy "untagged_resources_policy" do
     detail_template <<-EOS
 There are {{ len data }} resource(s) missing tags: {{ parameters.param_tag_keys }}
 
-| Cloud Name | Cloud Type | Name | ID | Type | State | Href | Missing Tags | Invalid Tags |
-| ---------- | ---------- | ---- | --- | ---- | ----- | ---- | ------------ | ------------ |
-{{ range data -}}
-| {{ .cloud_name }} | {{ .cloud_type }} | {{ .name }} | {{ .id }} | {{ .type }} | {{ .state }} | {{ .href }} | {{ range $i, $e := .missing_tags }}{{ if $i }}, {{ end }}{{ $e }}{{ end }} | {{ range $i, $e := .invalid_tags }}{{ if $i }}, {{ end }}{{ $e }}{{ end }}  |
-{{ end -}}
 EOS
+    export do
+      field "cloud_name" do
+        label "Cloud Name"
+      end
+      field "cloud_type" do
+        label "Cloud Type"
+      end
+      field "name" do
+        label "Name"
+      end
+      field "id" do
+        label "ID"
+      end
+      field "type" do
+        label "Type"
+      end
+      field "state" do
+        label "State"
+      end
+      field "href" do
+        label "Href"
+      end
+      field "range__i___e_____missing_tags" do
```

Comment:
Auto-conversion didn't work so well for this case -- will have to hand fix these type of things.
I think it should just be:
```
field "missing_tags" do
  label "Missing Tags"
end
```

These is untyped json data -- it can be boolean, float/int, string, an array, or a map. The UI will know how to display all those values except map is up in the air, I don't know what the UI might do with that. The UI should know to comma join arrays of simple types...